pipeline/07-export.R

```diff
@@ -413,8 +444,9 @@ assessment_pin_prepped <- assessment_pin_merged %>%
     sale_recent_1_outlier_type, sale_recent_1_document_num,
     sale_recent_2_date, sale_recent_2_price,
     sale_recent_2_outlier_type, sale_recent_2_document_num,
-    char_yrblt, char_total_bldg_sf, char_type_resd, char_land_sf,
-    char_apts, char_ncu,
+    char_yrblt, char_beds, char_ext_wall, char_bsmt, char_bsmt_fin, char_air,
+    char_heat, char_total_bldg_sf, char_type_resd, char_land_sf, char_apts,
+    char_ncu,
```

Comment:
Unfortunately `char_ext_wall` and `char_type_resd` are formatted using the short type rather than the long type in the PIN data, which is different from the way these columns are formatted in the card and training data. We could potentially do something similar to `format_char_apts` to reverse this for consistency, but I figure that isn't worth the effort and I wonder if you agree. The only pitfall that jumps out to me immediately is that `FRAM` (frame exterior) is different from `FRMA` (frame and masonry exterior) in the short codes for `char_ext_wall`, which could be misleading if a reader isn't paying close attention.

Reply:
I think it's okay. Those codes come directly from the AS/400 rendering of the numeric codes, so analysts will be very familiar with them.